src/NotesLoaderSSC.cpp

```diff
@@ -507,6 +514,15 @@ void SetStepsOffset(StepsTagInfo& info)
 		info.has_own_timing = true;
 	}
 }
+void SetStepsSyncBias(StepsTagInfo& info)
```

Comment:
This seems overkill, as mentioned above we really just need to subtract the 9ms if we detect the pack has ITG offset.